### ISSUE TITLE
TypeScript Error: 'zoom': boolean is not assignable to parameter of type 'GoogleMapOptions'

### ISSUE THREAD

**Issue by grayspace** [mapsplugin/cordova-plugin-googlemaps#1712]
**I'm submitting a ...**
[ ] question
[X] any problem or bug report
[ ] feature request

**The plugin version:**
[ ] **1.4.x**
[X] **2.0.0-beta3** 

**If you choose 'problem or bug report', please select OS:**
[X] **Android**
[ ] **iOS**

**cordova information:** 

```
cordova-plugin-compat 1.0.0 "Compat"
cordova-plugin-console 1.0.5 "Console"
cordova-plugin-device 1.1.4 "Device"
cordova-plugin-geolocation 2.4.3 "Geolocation"
cordova-plugin-googlemaps 2.0.0-beta3-20170901-1651 "cordova-plugin-g
ooglemaps"
cordova-plugin-splashscreen 4.0.3 "Splashscreen"
cordova-plugin-statusbar 2.2.2 "StatusBar"
cordova-plugin-whitelist 1.3.1 "Whitelist"
ionic-plugin-keyboard 2.2.1 "Keyboard"
```

**Current behavior:**
When calling map.setOptions, and attempting to set option 'zoom': true, in order to show zoom control on map, this throws a type definition error: 'zoom': boolean is not assignable to parameter of type 'GoogleMapOptions'.

**Expected behavior:**
Expected plugin to allow setting 'zoom': true in GoogleMapOptions so that the zoom control can be disabled or enabled as desired. Other controls, such as compass, myLocationButton, indoorPicker, and mapToolbar can all be configured from map.setOptions.

**Steps to reproduce:**
Setup map, then call map.setOptions:
```
this.map.setOptions({
    'controls': {
        'compass': true,
        'myLocationButton': false,
        'indoorPicker': false,
        'zoom': true, // <-- throws type definition error
        'mapToolbar': true
    }
});
```

**Possible solution:**
I was able to navigate to the GoogleMapOptions definition located at "node-modules\@ionic-native\google-maps\index.d.ts" and add "zoom? boolean;", which fixed the issue. Once this was added, I could set zoom true in map.setOptions and the zoom control showed on map without problems. Thanks!

**Comment by wf9a5m75** [comment]
Please show me your package.json

**Comment by wf9a5m75** [comment]
Right. By the way, the zoom controller is implemented on Android only. Nobody use the option anymore recently.

**Comment by wf9a5m75** [comment]
I sent a pull request to the ionic team.

**Comment by wf9a5m75** [comment]
The PR is merged. It will be released when the next version of the @ ionic-native/google-maps is released.

**Comment by wf9a5m75** [comment]
Sorry, after the discussion with the ionic team, I don't support the `@ionic-native/google-maps` anymore. Please ask to the ionic-native repository.

**Comment by wf9a5m75** [comment]
https://github.com/mapsplugin/cordova-plugin-googlemaps/issues/1749

**Comment by wf9a5m75** [comment]
Luckily, the fundraising is reached to the end quickly, I restart to support the @ionic-native/google-maps again.
I reopened the PR.

**Comment by wf9a5m75** [comment]
The PR is merged, and published as `@ionic-native/google-maps` ver 4.3.0.
Please reinstall the `@ionic-native/google-maps` plugin.
https://www.npmjs.com/package/@ionic-native/google-maps
